posthog/hogql_queries/experiments/test/test_funnels_statistics.py

```diff
@@ -45,6 +46,7 @@ def run_test_for_both_implementations(self, test_fn):
             calculate_credible_intervals=calculate_credible_intervals_v2,
         )
 
+    @flaky(max_runs=3, min_passes=1)
```

Comment:
small nit, you can add this to the class and then you don't have to add it for each test fn

Reply:
@neilkakkar Thanks for the tip! I think I like the level of explicitness having them at the method level, but I'll keep that in mind for the future.